nixos/modules/system/boot/systemd.nix

```diff
@@ -136,6 +139,16 @@ let
       "systemd-ask-password-wall.path"
       "systemd-ask-password-wall.service"
 
+      # Varlink APIs
+      "systemd-bootctl@.service"
+      "systemd-bootctl.socket"
```

Comment:
No specific docs: https://www.freedesktop.org/software/systemd/man/latest/bootctl.html is the main docs.

Source is https://raw.githubusercontent.com/systemd/systemd/main/units/systemd-bootctl.socket